### PR TITLE
SUMO-169401: seperated docs for sumologic_monitor_folder

### PR DESCRIPTION
**Pull request by rsanbhad**

### CODE REVIEW

File: website/docs/r/monitor_folder.html.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (sumovishal) [inline website/docs/r/monitor_folder.html.markdown:1]: We should remove example from https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/monitor#example-monitor-folder and point user to this new page.

File: website/docs/r/monitor_folder.html.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+- `type` - (Optional) The type of object model. Valid value:
+  - `MonitorsLibraryFolder`
+- `name` - (Required) The name of the monitor folder. The name must be alphanumeric.
+- `description` - (Required) The description of the monitor folder.
+- `parent_id` - (Optional) The ID of the Monitor Folder that contains this Monitor Folder. Defaults to the root folder.
+
+## Import
+
+Monitor folders can be imported using the monitor ID, such as:
```
Comment (sumovishal) [inline website/docs/r/monitor_folder.html.markdown:35]: "monitor ID" -> "monitor folder identifier"

File: website/docs/r/monitor_folder.html.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Argument reference
+
+The following arguments are supported:
+
+- `type` - (Optional) The type of object model. Valid value:
+  - `MonitorsLibraryFolder`
+- `name` - (Required) The name of the monitor folder. The name must be alphanumeric.
+- `description` - (Required) The description of the monitor folder.
+- `parent_id` - (Optional) The ID of the Monitor Folder that contains this Monitor Folder. Defaults to the root folder.
```
Comment (sumovishal) [inline website/docs/r/monitor_folder.html.markdown:31]: ID -> identifier

File: website/docs/r/monitor_folder.html.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+Provides the ability to create, read, delete, and update folders for [Monitors][1].
+
+## Example Monitor Folder
+
+NOTE: Monitor folders are considered a different resource from Library content folders.
+
+```hcl
+resource "sumologic_monitor_folder" "tf_monitor_folder_1" {
+  name        = "test terraform folder"
+  description = "a folder for monitors"
```
Comment (sumovishal) [inline website/docs/r/monitor_folder.html.markdown:19]: Add `parent_id` in the example?

**Review by sumovishal — APPROVED** [review]
lgtm, some minor comments.

File: website/docs/r/monitor_folder.html.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+layout: 'sumologic'
+page_title: 'SumoLogic: sumologic_monitor_folder'
+description: |-
+  Provides the ability to create, read, delete, and update folders for Monitors.
+---
+
+# sumologic_monitor
```
Comment (kumar-avijit) [inline website/docs/r/monitor_folder.html.markdown:8]: Should this say `sumologic_monitor_folder` instead?

**Review by kumar-avijit — APPROVED** [review]

**Pull request merged**